code/modules/client/preference_setup/loadout/loadout_belt.dm

```diff
@@ -3,12 +3,13 @@
 	path = /obj/item/storage/belt/generic
 	slot = slot_belt
 	sort_category = "Belts"
-	flags = GEAR_HAS_COLOR_SELECTION
+	flags = GEAR_HAS_NAME_SELECTION | GEAR_HAS_DESC_SELECTION | GEAR_HAS_COLOR_SELECTION
 
 /datum/gear/belt/fannypack
 	display_name = "fannypack"
 	cost = 2
 	path = /obj/item/storage/belt/fannypack/recolorable
+	flags = GEAR_HAS_COLOR_SELECTION
```

Comment:
That will make so that you can't rename the fanny pack.

Reply:
That was the intention as I didn't intend to make changes to how the fanny pack currently is (currently it only allows color selection), only to the basic belt.